web/styles/right_sidebar.css

```diff
@@ -364,6 +364,15 @@ $user_status_emoji_width: 24px;
         }
     }
 
+    #buddy-list-menu-icon {
+        opacity: 0.5;
+        justify-self: right;
```

Comment:
The `justify-self` here isn't necessary, as the `auto` column definition is just going to let the icon take up what space it will in the grid anyway.

Reply:
Removed, thanks